### ISSUE TITLE
ajax quantity single product

### ISSUE THREAD

**Issue by Cerimp** [themeszone/themeszone-woocommerce-ajax-quantity#1]
Hello
Very nice plugin, thanks!
tell me how to make an automatic update on a single product
For example here - http://plugins.themes.zone/product/happy-ninja/

**Comment by nehagrover027** [comment]
Yes I want the solution for same. Did you get any fix for this.? 
Plus how can I make it work on variable products.?
